config/initializers/omniauth.rb

```diff
@@ -26,4 +26,10 @@
     include_granted_scopes: true
   }
 
+  provider :microsoft_graph, Setting.microsoft_graph_auth_client_id, Setting.microsoft_graph_auth_client_secret, {
+    name: "microsoft_graph",
+    scope: Setting.microsoft_graph_scope.presence || %|
+      openid profile email offline_access user.read mailboxsettings.read
+    |
```

Comment:
Oh interesting, I like how you made the scopes part of the config. That’s a good improvement we could make to google, in the future.